### PR TITLE
feat: add missing getters

### PR DESCRIPTION
**Pull request by alvaro-ipor**

### CODE REVIEW

File: contracts/amm/MiltonInternal.sol [hunk 1]
```diff
@@ -309,14 +309,26 @@ abstract contract MiltonInternal is
         return _iporOracle;
     }
 
+    function getIporOracle() external view returns (address) {
+        return address(_iporOracle);
+    }
+
     function _getMiltonStorage() internal view virtual returns (IMiltonStorage) {
         return _miltonStorage;
     }
```
Comment (mario-ipor) [inline contracts/amm/MiltonInternal.sol:319]: On top of smart contract external methods on bottom internal and private - try to not mix visibility in listed methods like here external internal external

**Review by mario-ipor — CHANGES_REQUESTED** [review]

File: contracts/interfaces/IJosephInternal.sol [hunk 2]
```diff
@@ -12,6 +12,14 @@ interface IJosephInternal {
     /// @return ERC20 token address
     function getAsset() external view returns (address);
 
+    function getStanley() external view returns (address);
```
Comment (pete-ipor) [inline contracts/interfaces/IJosephInternal.sol:15]: add documentations comments

File: contracts/interfaces/IMiltonInternal.sol [hunk 3]
```diff
@@ -16,6 +16,12 @@ interface IMiltonInternal {
     /// @return asset's address
     function getAsset() external view returns (address);
 
+    function getIporOracle() external view returns (address);
+
```
Comment (pete-ipor) [inline contracts/interfaces/IMiltonInternal.sol:20]: add documentations comments

File: contracts/interfaces/IMiltonStorage.sol [hunk 4]
```diff
@@ -12,6 +12,10 @@ interface IMiltonStorage {
     /// @return current Milton Storage version, integer
     function getVersion() external pure returns (uint256);
 
+    function getMilton() external view returns (address);
```
Comment (pete-ipor) [inline contracts/interfaces/IMiltonStorage.sol:15]: add documentations comments

File: contracts/interfaces/IStanleyInternal.sol [hunk 5]
```diff
@@ -16,6 +16,8 @@ interface IStanleyInternal {
     /// @return Milton address
     function getMilton() external view returns (address);
 
+    function getIvToken() external view returns (address);
```
Comment (pete-ipor) [inline contracts/interfaces/IStanleyInternal.sol:19]: add documentations comments

**Review by pete-ipor — CHANGES_REQUESTED** [review]

**Review by mario-ipor — APPROVED** [review]

**Pull request merged**